includes/class-wcs-att-display.php

```diff
@@ -112,107 +103,9 @@ public static function show_subscription_options() {
-				$one_time_option_description = $product->is_type( 'variation' ) ? sprintf( __( '%1$s &ndash; %2$s', 'woocommerce-subscribe-all-the-things' ), $none_string, '<span class="price">' . WCS_ATT_Product::get_price_html( $product, false ) . '</span>' ) : $none_string;
-
-				$options[] = array(
-					'description' => apply_filters( 'wcsatt_single_product_one_time_option_description', $one_time_option_description, $product ),
-					'value'       => '0',
-					'selected'    => '0' === $default_subscription_scheme_option_value,
-					'data'        => apply_filters( 'wcsatt_single_product_one_time_option_data', array(), $product )
-				);
-			}
-
-			// Subscription options.
-			foreach ( $subscription_schemes as $subscription_scheme ) {
-
-				$sub_price_html = '<span class="price subscription-price">' . WCS_ATT_Product::get_price_html( $product, $subscription_scheme->get_key() ) . '</span>';
-
-				$option_data = array(
-					'subscription_scheme'   => $subscription_scheme->get_data(),
-					'overrides_price'       => $subscription_scheme->has_price_filter(),
-					'discount_from_regular' => apply_filters( 'wcsatt_discount_from_regular', false )
-				);
-
-				$options[] = array(
-					'description' => apply_filters( 'wcsatt_single_product_subscription_option_description', ucfirst( false === $force_subscription ? sprintf( __( '%s', 'product subscription selection - positive response', 'woocommerce-subscribe-all-the-things' ), $sub_price_html ) : $sub_price_html ), $sub_price_html, $subscription_scheme->has_price_filter(), false === $force_subscription, $product, $subscription_scheme ),
-					'value'       => $subscription_scheme->get_key(),
-					'selected'    => $default_subscription_scheme_option_value === $subscription_scheme->get_key(),
-					'data'        => apply_filters( 'wcsatt_single_product_subscription_option_data', $option_data, $subscription_scheme, $product )
-				);
-			}
-
-			if ( $prompt = WCS_ATT_Core_Compatibility::is_wc_version_gte_2_7() ? $product->get_meta( '_wcsatt_subscription_prompt', true ) : get_post_meta( $product_id, '_wcsatt_subscription_prompt', true ) ) {
-				$prompt = wpautop( do_shortcode( wp_kses_post( $prompt ) ) );
-			}
-
-			$options = apply_filters( 'wcsatt_single_product_options', $options, $subscription_schemes, $product );
-
-			ob_start();
-
-			wc_get_template( 'single-product/product-subscription-options.php', array(
-				'product'        => $product,
-				'product_id'     => $product_id,
-				'options'        => $options,
-				'allow_one_time' => false === $force_subscription,
-				'prompt'         => $prompt,
-			), false, WCS_ATT()->plugin_path() . '/templates/' );
-
-			$content = ob_get_clean();
-		}
-
-		return $content;
-	}
-
-	/**
-	 * Override the WooCommerce "Add to Cart" button text with "Sign Up Now".
-	 *
-	 * @since 1.1.1
-	 */
-	public static function add_to_cart_text( $button_text ) {
```

Comment:
Did you intend to remove this method also? The deprecation notice below is only for the `get_subscription_options_content()` method.